src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -9,14 +9,22 @@
 import seedu.address.model.student.Year;
 import seedu.address.model.student.admin.Admin;
 
+
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
```

Comment:
Should we remove the traces of address book?

Reply:
Good catch! I'll remove it in my next PR.